pkgs/development/python-modules/pyschemes/default.nix

```diff
@@ -1,24 +1,29 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-, pythonAtLeast
 , pytestCheckHook
+, fetchpatch
 }:
 
 buildPythonPackage rec {
   pname = "pyschemes";
   version = "unstable-2017-11-08";
   format = "setuptools";
 
-  disabled = pythonAtLeast "3.10";
-
   src = fetchFromGitHub {
     owner = "spy16";
     repo = pname;
     rev = "ca6483d13159ba65ba6fc2f77b90421c40f2bbf2";
     hash = "sha256-PssucudvlE8mztwVme70+h+2hRW/ri9oV9IZayiZhdU=";
   };
 
+  patches = [
+    # Fix python 3.10 compatibility
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/spy16/pyschemes/pull/6.patch";
```

Comment:
It is better to reference the commit itself because the PR patch can change afterwards, so the better url would be `https://github.com/spy16/pyschemes/commit/23011128c6c22838d4fca9e00fd322a20bb566c4.patch`

Reply:
good point! Fixed.